server/controllers/finance/journal/index.js

```diff
@@ -36,6 +40,64 @@ exports.journalEntryList = journalEntryList;
 
 exports.editTransaction = editTransaction;
 exports.count = count;
+exports.refenreceTransaction=refenreceTransaction;
+
+
+
+
+
+//loading identifiers keyies, used for defining the table name
+const identifiersIndex = {};
+function indexIdentifiers() {
+  _.forEach(identifiers, (entity) => {
+    identifiersIndex[entity.key] = entity;
+  });
+}
+
+// API GET /journal/refenrece_transaction/:codeRef/:language'
+
+//This function render a report in the browser
+//It takes a transaction reference code as paramter and the language
+//The reference code is a combination of table_key.project_abbr.reference
+//The table name is variable, it can be :invoice, cash or voucher
+
+function refenreceTransaction(req,res,next){
+   
+  //loading identifiers keyies, used for defining the table name
+  indexIdentifiers();
+
+  //reports urls
+  var reportsTypes={
+    VO:'/reports/finance/vouchers/',
+    IV:'/reports/finance/invoices/',
+    CP:'/reports/finance/cash/'
+  };
+
+  var codeRef =  req.params.codeRef.split(".");
+  const language= req.params.language;
+
+  const code = codeRef[0];
+  const project_name=codeRef[1];
+  const refence=codeRef[2];
```

Comment:
Please include a space between the name of the variable, the equals sign, and the assignment.  Like this:
```js
const refence = codeRef[2];
```

I also think this should probably be called `reference` instead of `refence`.